src/Components/SmartComponents/CVEs/CVEsTable.js

```diff
@@ -52,7 +52,7 @@ const CVEsTableWithContext = ({ context, header }) => {
                     rows={isEmpty ? noCves() : rows}
                     onSelect={!isEmpty ? handleOnSelect : undefined}
                     onCollapse={!isEmpty ? (event, rowKey) => methods.openCves(rowKey) : undefined}
-                    actionResolver={!(cves.data.length === 0) ?
+                    actionResolver={canEditStatusOrBusinessRisk && !(cves.data.length === 0) ?
```

Comment:
`!(cves.data.length === 0)` could be rewritten to `cves.data.length > 0` for simplificity